bob-apps/bin/bobd.rs

```diff
@@ -17,34 +17,62 @@ use std::fs::create_dir;
 #[macro_use]
 extern crate log;
 
+use log4rs::append::console::ConsoleAppender;
+use log4rs::encode::pattern::PatternEncoder;
+use log4rs::append::rolling_file::{
+    RollingFileAppender,
+    policy::compound::{
+        trigger::size::SizeTrigger,
+        roll::fixed_window::FixedWindowRoller,
+        CompoundPolicy
+    }
+};
+use log4rs::config::{Appender, Config, Logger, Root};
+
+use network_interface::{NetworkInterface, NetworkInterfaceConfig, Addr};
+
 #[tokio::main]
 async fn main() {
     let matches = get_matches();
 
-    if matches.value_of("cluster").is_none() {
-        eprintln!("Expect cluster config");
-        eprintln!("use --help");
-        return;
-    }
-
-    if matches.value_of("node").is_none() {
-        eprintln!("Expect node config");
-        eprintln!("use --help");
-        return;
-    }
+    let cluster;
+    let node;
+    if let (_, Some(sub_matches)) = matches.subcommand() {
```

Comment:
You should check subcommand name

Reply:
Fix: [3bdb33d](https://github.com/qoollo/bob/pull/588/commits/3bdb33d49a6cd64e060e5fc935e35767e4da086e)